includes/main.js

```diff
@@ -17,19 +17,67 @@ var pictures = [
 	'images/landscape-9.jpg',
 	'images/pexels-photo-132037.jpeg',
 	'images/pretty.jpg',
-];
+]; 
+
+
+
 
 function initiateApp(){
-	/*advanced: add jquery sortable call here to make the gallery able to be sorted
-		//on change, rebuild the images array into the new order
-	*/
-	makeGallery(pictures);
+	if (typeof(Storage) !== 'undefined' || typeof(Storage !== 'null')){
+
+		// localStorage.setItem("arraySort",JSON.stringify(pictures));
+	
+		stored = JSON.parse(localStorage.getItem("arraySort"));
+
+		console.log(stored);
+		console.log(typeof(stored));
+	}
+	else {
+		// var stored = pictures;
+	}
+		 /* advanced: add jquery sortable call here to make the gallery able to be sorted
+		 on change, rebuild the images array into the new order  */
+
 	addModalCloseHandler();
+
+	var sortableObj = { //?makes the .sortable obj that holds the events
+		start: makeGallery(stored),
```

Comment:
this wouldn't work quite the way you expect.  This runs the function immediately which is coincidentally the same time, but at worst this should be put into an anonymous function